examples/pytorch/igmc/data.py

```diff
@@ -0,0 +1,167 @@
+"""MovieLens dataset"""
```

Comment:
I believe many of our current examples use `MovieLens`. Could you check if they are based on the same data source? If so, can we have a unified interface for loading `MovieLens` and make it a built-in DGL dataset? I understand that different examples might involve different processing steps. Can we first have an issue summarizing the different processing steps different examples require?

Reply:
As far as I know, IGMC and GCMC use MovieLens dataset. Could you guide me to other examples that also use this dataset?

Reply:
I'm not quite sure what examples in DGL also use MovieLens dataset. It'd be much helpful to compare their utilization of MovieLens if you show me more such examples.

Reply:
You can search `MovieLens` in the repo. I also found `TAHIN` and `PinSAGE`.

Reply:
OK I see. Let me take a look. Thanks.

Reply:
I'm fine with that. Meanwhile, I'll prefer having a separate PR for MovieLens merged before we merge this PR.

Reply:
OK👌